src/appleseed.python/bindmasterrenderer.cpp

```diff
@@ -85,35 +88,33 @@ namespace
         }
 
         bpy::object                     m_project;
-        std::auto_ptr<MasterRenderer>   m_renderer;
+        std::unique_ptr<MasterRenderer> m_renderer;
     };
 
-    std::auto_ptr<MasterRendererWrapper> create_master_renderer(
+    std::shared_ptr<MasterRendererWrapper> create_master_renderer(
```

Comment:
Why `shared_ptr<>` here?

Reply:
We cannot pass unique_ptr to bpy::make_constructor because unique_ptr is non-copyable.